zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -282,8 +282,17 @@ mod test {
     };
     use bus_mapping::{evm::OpcodeId, mock::BlockData};
     use eth_types::{self, address, bytecode, evm_types::GasCost, geth_types::GethData, Word};
+    use lazy_static::lazy_static;
     use mock::TestContext;
 
+    lazy_static! {
+        static ref TEN_ETHER: Word = Word::from(10u64.pow(19));
+        static ref POINT_ONE_ETHER: Word = Word::from(10u64.pow(17));
+        static ref MINIMAL_GAS: Word =
+            Word::from(GasCost::TX.as_u64() + 2 * OpcodeId::PUSH32.constant_gas_cost().as_u64());
+        static ref TWO_GWEI: Word = Word::from(2_000_000_000);
+    }
+
```

Comment:
Can we instead add these things in mock crate? As they're used for tests only.

Also, I was thinking that we could add  a function macro to define Wei amounts as well as Ether ones. So that we don't need to hardcode every single value we need in tests.

Reply:
I just added helpers functions for `eth` and `gwei`.